server/src/firebase.ts

```diff
@@ -1,5 +1,39 @@
-import { initializeApp } from 'firebase-admin/app'
+import * as admin from 'firebase-admin/app'
 import { getFirestore } from 'firebase-admin/firestore'
 
-const firebaseApp = initializeApp()
+// Service Account info needed in emulator
+const serviceAccount = {
+    // local service account
+}
+
+// To fix type errors
+const key = serviceAccount as admin.ServiceAccount
+
+let firebaseApp
+
+const use_emulator = false
+if (use_emulator) {
+  // emulator in development
+  firebaseApp = admin.initializeApp({
+    credential: admin.cert(key),
+    databaseURL: 'http://localhost:8080?ns=ns=${project-id}', //project-id comes from your emulator     firebase projects:list
+  })
+} else {
+  // production
+  admin.initializeApp()
+}
+
 export const firestoreDb = getFirestore(firebaseApp)
+
+firestoreDb
```

Comment:
This line is not good - it will add this user everytime the server restarts, in live production db if this were deployed.